.github/workflows/install-slices.yaml

```diff
@@ -0,0 +1,121 @@
+name: Install slices
+
+on:
+  push:
+    branches:
+      - "main"
+    paths:
+      - ".github/**"
+  pull_request:
+    branches:
+      - "main"
+    paths:
+      - ".github/**"
+  schedule:
+    # Run at 00:00 every day.
+    # Ref: https://man7.org/linux/man-pages/man5/crontab.5.html
+    - cron: "0 0 * * *"
+  workflow_call:
+
+env:
+  # Package architectures and chisel-releases branches to test on.
+  ARCHES: ${{ toJson('["amd64","arm64","armhf","i386","ppc64el","riscv64","s390x"]') }}
+  RELEASES: ${{ toJson('["ubuntu-20.04","ubuntu-22.04","ubuntu-23.10","ubuntu-24.04"]') }}
+
+jobs:
+  prepare-install:
+    runs-on: ubuntu-latest
+    name: "Prepare to install"
+    outputs:
+      install-all: ${{ steps.set-output.outputs.install_all }}
+      matrix: ${{ steps.set-output.outputs.matrix }}
+    steps:
+      - name: Set output
+        id: set-output
+        run: |
+          set -ex
+          if [[
+            "${{ github.base_ref || github.ref_name }}" == "main" ||
+            "${{ github.event_name }}" == "schedule"
+          ]]; then
+            echo "matrix={\"ref\":${{ env.RELEASES }},\"arch\":${{ env.ARCHES }}}" >> $GITHUB_OUTPUT
+            echo "install_all=true" >> $GITHUB_OUTPUT
+          else
+            echo "matrix={\"arch\":${{ env.ARCHES }}}" >> $GITHUB_OUTPUT
+          fi
+
+  # The "install" job tests the slices by installing them.
+  # It installs **all** slices if:
+  #   - inputs.install-all is true
```

Comment:
not used anymore

Reply:
Thanks! Removed in https://github.com/canonical/chisel-releases/pull/119/commits/71a897eea58ac17332275feb6b252f1fb5d0931d.